tests/server.py

```diff
@@ -367,8 +367,6 @@ def setup_test_homeserver(cleanup_func, *args, **kwargs):
     """
     server = _sth(cleanup_func, *args, **kwargs)
 
-    database = server.config.database.get_single_database()
```

Comment:
As far as I can tell this isn't doing anything special (like causing the database to get cached or something).

Reply:
agreed. I came across it the other day too.